From ce165480cf25584bd48a3daf5904e07999f568a8 Mon Sep 17 00:00:00 2001
From: David Barnett <davidb@rnett.nz>
Date: Fri, 10 May 2024 16:51:04 +1000
Subject: [PATCH] patch: remove chdir

---
 main.go | 2 --
 1 file changed, 2 deletions(-)

diff --git main.go main.go
index 6b5586c..e0bb67a 100644
--- main.go
+++ main.go
@@ -46,8 +46,6 @@ func main() {
 	cmd.Run = func(cmd *cli.Command, args []string) error {
 		config := loadConfig(*configFile)
 
-		os.Chdir(filepath.Dir(*configFile))
-
 		for _, t := range config.Specs {
 			if len(args) > 0 && !hasStr(args, t.Output) {
 				continue
-- 
2.39.3 (Apple Git-146)

