From 5da8d3d39c5688f25bd2d05cf4ceaef4ce8e5bce Mon Sep 17 00:00:00 2001
From: David Barnett <davidb@rnett.nz>
Date: Sun, 16 Jun 2024 14:36:01 +1000
Subject: [PATCH] remove recursive mixin

---
 pkg/render/render.go | 5 -----
 1 file changed, 5 deletions(-)

diff --git pkg/render/render.go pkg/render/render.go
index a8670c9..ef6719e 100644
--- pkg/render/render.go
+++ pkg/render/render.go
@@ -139,11 +139,6 @@ func Kind(name string, k model.Kind) j.ObjectType {
 
 	SortFields(fields)
 
-	// mixin field for compatibility (patch to avoid recursive result)
-	fields = append(fields,
-		j.String("#mixin", "ignore"),
-		j.Ref("mixin", "self"),
-	)
 	return j.Object(name, fields...)
 }
 
-- 
2.39.3 (Apple Git-146)

