From 7e7e0d57ed5e968d6779aca46e8298d1ccc23c31 Mon Sep 17 00:00:00 2001
From: David Barnett <davidb@rnett.nz>
Date: Mon, 8 Jul 2024 21:08:49 +1000
Subject: [PATCH] disable printing

---
 main.go | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git main.go main.go
index 6b5586c..3fe76e4 100644
--- main.go
+++ main.go
@@ -57,7 +57,7 @@ func main() {
 
 			if len(t.Crds) > 0 {
 				for _, url := range t.Crds {
-					log.Printf("Generating '%s' from '%s, %s'", t.Output, url, t.Prefix)
+					// log.Printf("Generating '%s' from '%s, %s'", t.Output, url, t.Prefix)
 
 					defs, err := swagger.Load(&swagger.CRDLoader{}, url)
 					if err != nil {
@@ -69,7 +69,7 @@ func main() {
 				}
 
 			} else {
-				log.Printf("Generating '%s' from '%s, %s'", t.Output, t.Openapi, t.Prefix)
+				// log.Printf("Generating '%s' from '%s, %s'", t.Output, t.Openapi, t.Prefix)
 
 				d, err := swagger.Load(&swagger.SwaggerLoader{}, t.Openapi)
 				if err != nil {
-- 
2.39.3 (Apple Git-146)

